docs/projects/ad777x_fmcz/index.rst

```diff
@@ -48,15 +48,6 @@ Supported carriers
    * - EVAL-AD7779-ARDZ
      - `ZedBoard <https://digilent.com/shop/zedboard-zynq-7000-arm-fpga-soc-development-board>`__
      - PMOD-JA, PMOD-JB, PMOD-JC
-   * - EVAL-AD7770-ARDZ
-     - :intel:`DE10-Nano <content/www/us/en/developer/topic-technology/edge-5g/hardware/fpga-de10-nano.html>`
-     - Arduino shield connector
-   * - EVAL-AD7771-ARDZ
-     - :intel:`DE10-Nano <content/www/us/en/developer/topic-technology/edge-5g/hardware/fpga-de10-nano.html>`
-     - Arduino shield connector
-   * - EVAL-AD7779-ARDZ
-     - :intel:`DE10-Nano <content/www/us/en/developer/topic-technology/edge-5g/hardware/fpga-de10-nano.html>`
-     - Arduino shield connector
```

Comment:
@PopPaul2021, would we want to mention that we had support for DE10-Nano, and say it's retired and give link to the hdl_2023_r2 release where it is last found?